homeassistant/components/gpsd/sensor.py

```diff
@@ -43,6 +48,35 @@
 
 DEFAULT_NAME = "GPS"
 
+
+class FixMode(StrEnum):
+    """Fix mode states."""
+
+    FIX_2D = "2d_fix"
+    FIX_3D = "3d_fix"
+
```

Comment:
Previously, I was thinking of something like this:
```suggestion
_MODE_VALUES = {2: FixMode.FIX_2D, 3: FixMode.FIX_3D}
```
So the dict doesn't need to be created each time.
Not sure if you still want to leave the enum.